### PR TITLE
Some minor corrections in the alerts documentation

### PR DESCRIPTION
**Pull request by aruniiird**

### CODE REVIEW

File: alerts/openshift-container-storage-operator/ObcQuotaBytesExhausedAlert.md [hunk 1]
```diff
@@ -15,7 +15,6 @@ OBC has exhausted and reached it's limit.
 Alert message will clearly indicate which OBC has reached the quota bytes limit.
 Look at the deployments attached to the OBC and see what all apps are
 using/filling-up the OBC.
-For quota help please visit: https://access.redhat.com/articles/6541861
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/ObcQuotaBytesExhausedAlert.md:18]: Why this link is not useful?

**Review by jmolmo — CHANGES_REQUESTED** [review]
Just wanted to know why the link you have removed is not useful.  
https://access.redhat.com/articles/6541861
"Quota support for OBC provided by RGW - Developer Preview" 

If this is not the right document... do we have another document (or section in official documentation) where is explained how to deal with OBC's  capacity problems?

File: alerts/openshift-container-storage-operator/ObcQuotaBytesExhausedAlert.md (hunk 1 above)
Comment (aruniiird) [reply alerts/openshift-container-storage-operator/ObcQuotaBytesExhausedAlert.md:18]: This is the comment I got in this excel sheet: https://docs.google.com/spreadsheets/d/1Gdx9nn4LNMMe-z-4TWFNDA_x3GN48Q6Rlk5yttjQ2MY/edit#gid=0
`sublink leads to dev preview article - https://access.redhat.com/articles/6541861`
and I don't have any other link to support

Comment (jmolmo) [reply alerts/openshift-container-storage-operator/ObcQuotaBytesExhausedAlert.md:18]: Well i think that to include the information about how to increase quotas will be for sure useful. 
What if you replace the "link" by the text in "Setting quota options by using oc command (No UI support, specific to OBCs from RGW, not for MCG)" of the document.?
In this way we provide the information to update properly the quota, making the user life easier.

Comment (aruniiird) [reply alerts/openshift-container-storage-operator/ObcQuotaBytesExhausedAlert.md:18]: Thanks @jmolmo for the details. Added the needed info. Please review

**Review by jmolmo — APPROVED** [review]
just fix the lint issues

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Pull request merged**